_data/numerics.yaml

```diff
@@ -986,7 +988,7 @@ values:
         numeric: "270"
         origin: InspIRCd 2.0
         format: "<client> :<count> servers and <count> users, average <average count> users per server"
-        conflict: true
```

Comment:
Hmm, we tend to remove the `conflict: true` line from the other `270` numeric when one's marked as obsoleted. Should we do that here, or is there a reason to leave the other one as saying conflict?

Reply:
Fixed.

Reply:
Awesome, thanks very much!